app/(tabs)/index.tsx

```diff
@@ -132,7 +134,9 @@ const Home = () => {
         />
       </View>
       <View className="flex-row items-center w-full justify-between px-6 mb-3">
-        <Text className="text-[20px] font-['Urbanist-Bold']">Top Doctors</Text>
+        <Text className="text-[20px] font-['Urbanist-Bold']"  onPress={() => {
+                router.push("/Doctors/topDoctors");
+              }}>Top Doctors</Text>
```

Comment:
This link to top doctors should be on the `See All` text right next to `Top Doctors`

Reply:
Okay I was a little confused on where to link it .Let me change it